model/flow/sealing_segment.go

```diff
@@ -87,10 +91,10 @@ func (segment *SealingSegment) Lowest() *Block {
 
 // FinalizedSeal returns the seal that seals the lowest block.
 // Per specification, this seal must be included in a SealingSegment.
-// Under normal operations (where only a validated SealingSegment
-// is processed), no error returns are expected.
+// The receiver SealingSegment must be validated.
```

Comment:
What is meant by "receiver sealing segment " ?

Reply:
The receiver is the name for the parameter specified in the first set of braces.
```go
func (r Receiver) DoSomething(...)
```

Reply:
Gotcha , "The receiver" sounds like it is something related to the sealing segment. +1 for removing it.